src/components/PeopleList.js

```diff
@@ -1,50 +1,38 @@
 import { GoHeart, GoHeartFill } from 'react-icons/go';
-import { useState, useEffect } from 'react';
 
-function PeopleList({ starWarsChars }) {
-    const [toggleFavourite, setToggleFavourite] = useState({});
-
-    useEffect(() => {
-        const savedFavourite = localStorage.getItem("toggleFavourite");
```

Comment:
Itt szerintem a toggleFavourite a local storage-ben nem egészen reprezentálja, hogy mit is tárolsz benne, mert egy action-nek tűnik a "toggle" miatt. Szóval azt szerintem érdemes lenne átnevezni :)

Reply:
A "savedFavourite" is picit megtévesztő változónév, hiszen egy valamire utal, pedig többet tudok elmenteni